src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -526,11 +526,17 @@ extern_crate_item ::= EXTERN CRATE IDENTIFIER alias? ';'
 
 mod_decl_item ::= MOD IDENTIFIER ';'
 
-mod_item ::= MOD IDENTIFIER '{' inner_attr* docs_and_item* '}' {
-  pin(".*") = 3
+mod_item ::= MOD IDENTIFIER mod_body {
+  methods = [
+    // FIXME(jajakobyly): This method is tagged as @Nullable while we want @NotNull
+    itemList = "/mod_body/item_"
+  ]
```

Comment:
The overall problem here is, that `file_mod_item` extends `mod_item` but the former doesn't have anything like a `mod_body`. I decided to unify their apis (so users do not have to differentiate between mod and file mod) by creating artificial `getItemList` method on `mod_item`.